Misc/NEWS.d/next/Core and Builtins/2020-05-19-19-39-49.bpo-40679.SVzz9p.rst

```diff
@@ -0,0 +1 @@
+Certain error messages about too few or too many arguments now refer to a function by its :term:`qualified name`.
```

Comment:
This wording can be simplified a little: "... now include the function's `term:qualified name`."